yampl.pyx

```diff
@@ -24,7 +24,7 @@ cdef extern from "ISocket.h" namespace "yampl":
 	cdef cppclass ISocket:
 		void send(void *, size_t) except +
 		size_t recv(void *&buffer) except +
-		ssize_t tryRecv(void *&buffer) except +
+		ssize_t tryRecv(void *&buffer,long timeout) except +
```

Comment:
This isn't backwards compatible. Please ensure that the public API isn't changed; you could for example use a default value for the `timeout` parameter.